Modules/Tokens/test/Tokens.API.Tests.Integration/StepDefinitions/TokensApiStepDefinitions.cs

```diff
@@ -116,7 +116,7 @@ public async Task GivenTheUserCreatedMultipleTokens()
             var createTokenRequest = new CreateTokenRequest
             {
                 Content = "QQ==",
-                ExpiresAt = DateTime.Now.AddDays(1).ToString("dd-MM-yyyy")
+                ExpiresAt = DateTime.Now.AddDays(1).ToString("MM-dd-yyyy")
```

Comment:
Can you try to use `yyyy-MM-dd` as format? That should do it as well and looks better imo